src/main/java/org/isf/video/gui/PhotoboothComponentImpl.java

```diff
@@ -241,36 +235,27 @@ public void actionPerformed(final ActionEvent actionEvent) {
         };
         getOkButton().setAction(okAction);
 
-        getCancelButton().addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent actionEvent) {
-                PhotoboothComponentImpl.this.presentationModel().triggerFlush();
-                PhotoboothComponentImpl.this.cleanup();
-                owner.dispose();
-            }
+        getCancelButton().addActionListener(actionEvent -> {
```

Comment:
also `actionEvent` is a good candidate as parameter for all these cases (just a note)

Reply:
OK; I'll see what I can do